Model/Reclaim.php

```diff
@@ -106,6 +113,59 @@ public function product($quote_id, $item_id) {
         return $response;
     }
 
+    /**
+    * @return mixed
+    */
+    public function productVariantInventory($product_id, $store_id=0)
+    {
+        if (!$product_id){
+            throw new NotFoundException(_('A product id is required'));
+        }
+        // if store_id is specificed, use it
+        if ($store_id){
+            $product = $this->_productFactory->create()->setStoreId($store_id)->load($product_id);    
+        } else {
+            $product = $this->_productFactory->create()->load($product_id);
+        }
+
+        if (!$product){
+            throw new NotFoundException(_('A product with id '. $product_id .' was not found'));
+        }
+
+        $productId = $product->getId();
```

Comment:
If `$product_id` is provided in the signature why do you need to get it again? Is this value different?